ha/core/event_manager/event_manager.py

```diff
@@ -383,27 +384,28 @@ def get_events(self, component : SUBSCRIPTION_LIST) -> list:
                     break
         return value
 
-    def publish(self, event: RecoveryActionEvent) -> None:
+    def publish(self, event: Event) -> None:
```

Comment:
As discussed please update the LLD (fault tolerance and Action Status Update LLD) since the name "RecoveryActionEvent" is mentioned in both and now we are using "Event" class/schema instead

Reply:
sure, will do that

Reply:
Done @ArchanaLimaye